reco_utils/recommender/vae/sparse_vae_2.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pandas as pd
+import numpy as np
+import itertools
+
+from scipy.sparse import coo_matrix
+import logging
+
+# import default parameters
+from reco_utils.common.constants import (
+    DEFAULT_USER_COL,
+    DEFAULT_ITEM_COL,
+    DEFAULT_RATING_COL,
+    DEFAULT_TIMESTAMP_COL,
+    DEFAULT_PREDICTION_COL,
+)
+
+
+log = logging.getLogger(__name__)
+
+
+def binarize_matrix(data, threshold):
+    """Takes a matrix as an input, and binarizes its entries based on a certain threshold.
+    
+    Args: 
+        data (np.array, float): the matrix of user-item interaction with ratings.
+        threshold (float): the threshold for binarization.
+        
+    Returns:
+        data_ (np.array, int): the click matrix.
+    """
+    data_ = data.copy()
+    data_[data_ < threshold] = 0
+    data_[data_ > threshold] = 1
+    
+    return data_
```

Comment:
can you please check whether this function https://github.com/microsoft/recommenders/blob/837b8081a4421e144f2bc05ba949c5ac6c52320f/reco_utils/common/python_utils.py#L106 is not doing the same operation?

Reply:
Yes, they actually do the same thing. I could not find this function earlier. We are going to replace our binarize_matrix function everywhere by already existing binarize function.

Reply:
I uploaded the requested changes. Please let us know if you have any comments. Also, we are available for a call if you think this will help us do extra progress.